parser-typechecker/src/Unison/UnisonFile/Type.hs

```diff
@@ -17,7 +17,7 @@ import Unison.WatchKind (WatchKind)
 data UnisonFile v a = UnisonFileId
   { dataDeclarationsId :: Map v (TypeReferenceId, DataDeclaration v a),
     effectDeclarationsId :: Map v (TypeReferenceId, EffectDeclaration v a),
-    terms :: Map v (a {- ann for whole binding -}, Term v a),
+    terms :: Map v (a {- ann for name of the binding -}, Term v a),
```

Comment:
Oooh, I hope changing this here cause surprises later; e.g. if we want to slice up and reorder the scratch file, we won't be able to use this annotation to do it if it's just the name.

Reply:
See the auto-format transcripts which do exactly what you suggest (that is, they rewrite the scratch file by replacing each function span with the formatted text of that function)

Reply:
Ah great thanks